docs/run-taco.md

```diff
@@ -9,9 +9,16 @@ Starting in 2019.4 Beta 1, you can load packaged connectors (otherwise known as
 1. Launch Tableau Desktop.
 
 ## Run a packaged connector in Tableau Server
-1. Create a directory for Tableau connectors. For example:   
+### Option 1
```

Comment:
Option 1 I think is likely the preferred path.  Option 2 is basically the developer and non-packaged, but seems good to document both in the SDK at least.